### ISSUE TITLE
Should clear the output screen between runs

### ISSUE THREAD

**Issue by GoogleCodeExporter** [jsvazic/double-fanucci-ga#2]
```
What steps will reproduce the problem?
1. Run the calculator a few times

What is the expected output? What do you see instead?
Expected: Fresh output per run
Actual: Results are appended to the end of the previous output.

```

Original issue reported on code.google.com by `jsva...@gmail.com` on 8 May 2009 at 6:30

**Comment by GoogleCodeExporter** [comment]
Original comment by `jsva...@gmail.com` on 8 May 2009 at 6:44
- Added labels: **Milestone-Release1.1**

**Comment by GoogleCodeExporter** [comment]
Original comment by `jsva...@gmail.com` on 9 May 2009 at 3:19
- Changed state: **Started**

**Comment by GoogleCodeExporter** [comment]
```
Fixed for v1.1
```

Original comment by `jsva...@gmail.com` on 9 May 2009 at 5:14
- Changed state: **Fixed**
